aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -439,19 +446,20 @@ Resources:
       ReceiveMessageWaitTimeSeconds: 20
       VisibilityTimeout: 15
       QueueName: <%="activities_dead-#{stack_name}"%>
+<% end -%>
 <%
-   if environment == :adhoc
+   if environment == :adhoc || stack_name == 'staging-next'
    self.daemon = true
 -%>
-  Daemon:
+  WebServer:
```

Comment:
Use the same logical-id as the standalone stack, to avoid replacing the resource.